### ISSUE TITLE
Why hardcode the maximum number of menu items?

### ISSUE THREAD

**Issue by hapoza** [vtex-apps/category-menu#43]
## Expected Behavior
Some stores have big menus with 8 or 10 or more items. We should be able to configure what's the maximum.

## Current Behavior
There's a hardcoded limit of 6 items, see https://github.com/vtex-apps/category-menu/blob/9caf949ebeaa6d16f1966d8c9f900f3caf783fbc/react/index.js#L14

## Possible Solution
Put the maximum number of items as a property on the schema as a layout option, or allow sending the maximum value by props.

## Context
The layout the client wants cannot be done in the current state of the app
![screenshot at jan 15 14-48-58](https://user-images.githubusercontent.com/27775611/51195734-d52adc00-18d4-11e9-8fa8-694c831d89dc.png)

**Comment by rerissondaniel** [comment]
Hi, thanks for opening the issue. That will be addressed in [this PR] (https://github.com/vtex-apps/category-menu/pull/45). We will try to find a way to make storefront feed itself, but if that is not possible we will simply remove that limitation.

**Comment by rerissondaniel** [comment]
Hi @hapoza, could you please check if that solves your problem?
We are planning to do a refactor of that component soon (when storefront comes back), that will make it easier to customize and build a menu like the one your client wants.

**Comment by hapoza** [comment]
I think it would, but it was release for the 2.x version, and dreamstore@1.x uses the version 1.x of this app. I was strictly forbidden from using dreamstore@2.x by @brunojdo until further notice.

On the project I was doing when I created the issue, I had to fork over this app and have it internal to the project and "solved" what I needed.

But I have another project coming up that also has more than 6 menu items and I hope dreamstore@2.x will be ready, and I'll let you know here if it works as intended 👍
